subsys/bluetooth/host/conn.c

```diff
@@ -1553,6 +1553,27 @@ void bt_conn_process_tx(struct bt_conn *conn)
 	}
 }
 
+bool bt_conn_exists_le(u8_t id, const bt_addr_le_t *peer)
+{
+	struct bt_conn *conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, peer);
```

Comment:
This function takes `u8_t id`, yet you pass `BT_ID_DEFAULT` to `bt_conn_lookup_addr_le()`. Is that intentional?

Reply:
Not intentional, will fix. Thank you.